frontend/src/classes/Player.ts

```diff
@@ -1,44 +1,44 @@
-export default class Player {
-  public location?: UserLocation;
-
-  private readonly _id: string;
-
-  private readonly _userName: string;
-
-  public sprite?: Phaser.GameObjects.Sprite;
-
-  public label?: Phaser.GameObjects.Text;
-
-  constructor(id: string, userName: string, location: UserLocation) {
-    this._id = id;
-    this._userName = userName;
-    this.location = location;
-  }
-
-  get userName(): string {
-    return this._userName;
-  }
-
-  get id(): string {
-    return this._id;
-  }
-
-  static fromServerPlayer(playerFromServer: ServerPlayer): Player {
-    return new Player(playerFromServer._id, playerFromServer._userName, playerFromServer.location);
-  }
-}
-
-export type ServerPlayer = {
-  _id: string;
-  _userName: string;
-  location: UserLocation;
-};
-
-export type Direction = 'front' | 'back' | 'left' | 'right';
-
-export type UserLocation = {
-  x: number;
-  y: number;
-  rotation: Direction;
-  moving: boolean;
-};
```

Comment:
Confused as to what's going on with this change? Doesn't even look like a lint, since the whitespace is the same

Reply:
apparently it is just a whitespace change—something to do with the number of spaces in the indents. linter loves to change things